pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -46,14 +54,26 @@ let
     pname = "paperless-ngx-frontend";
     inherit version src;
 
-    npmDepsHash = "sha256-uDaZ7j7IDgKy7wCWND2xzR1qHwUtdyjR4eyIAVy01dM=";
+    npmDepsHash = "sha256-K7wTYGGwEhPoXdRD+4swhSlMH0iem6YkF0tjnVHh7K8=";
 
     nativeBuildInputs = [
       python3
     ] ++ lib.optionals stdenv.isDarwin [
       xcbuild
     ];
 
+    buildInputs = [
+      cairo
+      giflib
+      libjpeg
+      libpng
+      librsvg
+      pango
+      pkg-config
+    ] ++ lib.optionals stdenv.isDarwin [
+      mupdf
```

Comment:
Not sure where this is coming from

Reply:
Had some errors around `mutool` on aarch64-Darwin. But I'll reevaluate to see if I need this after putting pkg-config in the right place.

Reply:
Looks like this is actually an error with the `img2pdf` python package building on aarch64-darwin...

Here's what I get on my machine:

```
nixpkgs on paperless-ngx-210-bump [$] took 2m39s
❯ nix build path:(pwd)#python311Packages.img2pdf
error: builder for '/nix/store/9zks1lvjp7vb573bc7977x1dfhwnpk0i-python3.11-img2pdf-0.5.1.drv' failed with exit code 2;
       last 10 log lines:
       > /nix/store/ai4hd8f1xhr0rfjdr17bxx1rwi42sx97-python3-3.11.6/lib/python3.11/subprocess.py:466: in check_output
       >     return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
       > /nix/store/ai4hd8f1xhr0rfjdr17bxx1rwi42sx97-python3-3.11.6/lib/python3.11/subprocess.py:571: in run
       >     raise CalledProcessError(retcode, process.args,
       > E   subprocess.CalledProcessError: Command '['mutool', '-v']' died with <Signals.SIGABRT: 6>.
       > =========================== short test summary info ============================
       > ERROR src/img2pdf_test.py - subprocess.CalledProcessError: Command '['mutool', '-v']' died with <Signal...
       > !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
       > =============================== 1 error in 5.12s ===============================
       > /nix/store/2a1pjx5crgygz7wbvl1lmp49n6apfp8r-stdenv-darwin/setup: line 1568: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/9zks1lvjp7vb573bc7977x1dfhwnpk0i-python3.11-img2pdf-0.5.1.drv'.
```

Reply:
And looks like it's actually a `mupdf` error:

```
nixpkgs on  paperless-ngx-210-bump [$] took 2m12s
❮ nix build path:(pwd)#mupdf

nixpkgs on  paperless-ngx-210-bump [$] took 2m12s
❯ result-bin/bin/mutool -v
dyld[85912]: Library not loaded: build/shared-release/libmupdf.dylib
  Referenced from: <no uuid> /nix/store/53anxhdg9mqh8cj568hrqd8w8yfxp07f-mupdf-1.23.5-bin/bin/mutool
  Reason: tried: 'build/shared-release/libmupdf.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OSbuild/shared-release/libmupdf.dylib' (no such file), 'build/shared-release/libmupdf.dylib' (no such file), '/usr/local/lib/libmupdf.dylib' (no such file), '/usr/lib/libmupdf.dylib' (no such file, not in dyld cache)
fish: Job 1, 'result-bin/bin/mutool -v' terminated by signal SIGABRT (Abort)
```

Reply:
I'm not sure I have the firepower to debug this issue in a timely manner. Should it be a blocker for merging this PR?

Reply:
Opened issue #272789 for this.

Reply:
If the relevant package was broken like this before, we can skip this for now.

Reply:
Looks like this will also be resolved with this pr: https://github.com/NixOS/nixpkgs/pull/272807